File: AppGR/FontEnd/app/src/main/java/com/example/appgrit/models/EditUserImageModel.java
package com.example.appgrit.models;

public class EditUserImageModel {
    private String UserId;
    private String ImagePath;

    // Constructors, getters, and setters

    public EditUserImageModel() {
    }

    public EditUserImageModel(String userId, String imagePath) {
        UserId = userId;
        ImagePath = imagePath;
    }

    public String getUserId() {
        return UserId;
    }

    public void setUserId(String userId) {
        UserId = userId;
    }

    public String getImagePath() {
        return ImagePath;
    }

    public void setImagePath(String imagePath) {
        ImagePath = imagePath;
    }
}

File: AppGR/BackEnd/AppGrIT/Models/EditUserImageModel.cs
using System.ComponentModel.DataAnnotations;

namespace AppGrIT.Models
{
    public class EditUserImageModel
    {
        [Required]
        public string UserId
        { get; set; } = null!;

        [Required]
        public string ImagePath
        { get; set; } = null!;


    }
}
